src/main/scala/nl.knaw.dans.easy.dd2d/DepositIngestTask.scala

```diff
@@ -51,28 +48,29 @@ case class DepositIngestTask(deposit: Deposit, dansBagValidator: DansBagValidato
     logger.info(s"Ingesting $deposit into Dataverse")
 
     for {
-      validationResult <- dansBagValidator.validateBag(bagDirPath)
-      _ <- Try {
-        if (!validationResult.isCompliant) throw RejectedDepositException(deposit,
-          s"""
-             |Bag was not valid according to Profile Version ${ validationResult.profileVersion }.
-             |Violations:
-             |${ validationResult.ruleViolations.map(_.map(formatViolation).mkString("\n")).getOrElse("") }
-          """.stripMargin)
-      }
+      //      validationResult <- dansBagValidator.validateBag(bagDirPath)
+      //      _ <- Try {
+      //        if (!validationResult.isCompliant) throw RejectedDepositException(deposit,
+      //          s"""
+      //             |Bag was not valid according to Profile Version ${ validationResult.profileVersion }.
+      //             |Violations:
+      //             |${ validationResult.ruleViolations.map(_.map(formatViolation).mkString("\n")).getOrElse("") }
+      //          """.stripMargin)
+      //      }
       ddm <- deposit.tryDdm
       dataverseDataset <- mapper.toDataverseDataset(ddm, deposit.vaultMetadata)
       json = Serialization.writePretty(dataverseDataset)
```

Comment:
Why first serialize and then extract again?

Reply:
Fixed. I removed the debug statement as well (since there is no need to serialize). Do we need to see the Dataset json?